### PR TITLE
feat: NFT #Number not showing in profiles

### LINKED ISSUE #9144: NFT #Number not showing in profiles
Why I do not see the numbers of generative NFTs I collected in my profile?

Profile - Collected - check the NFTs.

All those NFTs have # number - we should show that too

Screenshot 2024-01-26 at 12 44 26

Screenshot 2024-01-26 at 12 44 34

## Video

https://github.com/kodadot/nft-gallery/assets/104380776/88873eea-3c0b-4cf7-b9d7-9b933b124f16

Pls add them, we should show the number/edition/ of the NFT

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

  👇 __ Let's make a quick check before the contribution.

  ## PR Type

  - [x] Feature

  ## Needs QA check

  - @kodadot/qa-guild please review

  ## Context

  - [x] Closes #9144

  #### Did your issue had any of the "$" label on it?

  - [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

  ## Screenshot 📸

  - [x] My fix has changed UI

image

  ## Copilot Summary
  copilot:summary

  copilot:poem

### CODE REVIEW

File: components/shared/nftCard/NftMediaInfo.vue [hunk 1]
```diff
@@ -52,13 +53,25 @@ const props = withDefaults(
...
     variant?: NftCardVariant
+    displayNameWithSn?: boolean
   }>(),
   {
     collectionPopoverShowDelay: 500,
     variant: 'primary',
   },
 )
 
+const nameWithSn = computed(() => {
```
Comment (roiLeo) [inline components/shared/nftCard/NftMediaInfo.vue:64]: I would rename it 'name' since it doesnt Always has serial number

**Review by roiLeo — APPROVED** [review]
Otherwise lgtm

**Comment by roiLeo** [comment]
Btw there are many issue with showing serial number that could have same implementation

**Comment by Jarsen136** [comment]
> Btw there are many issue with showing serial number that could have same implementation

Yup, that should be the reason why we do not apply them on the items explorer page.

File: components/shared/nftCard/NftMediaInfo.vue (hunk 1 above)
Comment (Jarsen136) [reply components/shared/nftCard/NftMediaInfo.vue]: done

**Comment by yangwao** [comment]
Thanks!
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 6.73 USD/DOT ~ 4.458 $DOT
🧗 [16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z](https://polkadot.subscan.io/account/16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)
🔗 [0xe851a76b5d2241427a4169c828457054604926f7e3ccd7c8239d11be61fef1ab](https://polkadot.subscan.io/extrinsic/0xe851a76b5d2241427a4169c828457054604926f7e3ccd7c8239d11be61fef1ab)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**